### PR TITLE
Add .NET Core and Standard JS Tests Documentation

### PR DESCRIPTION
**Pull request by armanio123**
Added documentation for JavaScript and TypeScript unit tests support on .NET Core and .NET Standard.

The feature is yet to be released so please avoid merging this pull request until is publicly available. I can chime in when that happens.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@armanio123 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by armanio123** [comment]
Mentioning @amcasey for review.

File: docs/javascript/unit-testing-javascript-with-visual-studio.md [hunk 1]
```diff
@@ -171,5 +171,11 @@ To enable this, right-click the project node in the Solution Explorer, choose **
 Next, add your tests to the test root folder you specified, and they will be available to run in the
 Test Explorer window. If they don't initially appear, you may need to rebuild the project.
 
-> [!NOTE]
-> This currently doesn't work for .NET Standard and .NET Core projects.
+### Unit test .NET Core and .NET Standard
+Additional to the properties above, you will also need to install the NuGet package [Microsoft.JavaScript.UnitTest](https://www.nuget.org/packages/Microsoft.JavaScript.UnitTest/) and the property:
```
Comment (amcasey) [inline docs/javascript/unit-testing-javascript-with-visual-studio.md]: "Additional to" -> "In addition to"
"and the property" -> "and set the property"

**Comment by PRMerger19** [comment]
@armanio123 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/javascript/unit-testing-javascript-with-visual-studio.md (hunk 1 above)
Comment (armanio123) [reply docs/javascript/unit-testing-javascript-with-visual-studio.md]: Fixed.

**Comment by Jak-MS** [comment]
@Mikejo5000 - you're the author on record. can you take a look? thanks!

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by armanio123** [comment]
Please don't merge yet this PR. We'll need to wait until this feature is publicly available on Visual Studio.

**Comment by Mikejo5000** [comment]
#hold-off Holding off the merge

**Comment by ktoliver** [comment]
@Mikejo5000 @armanio123 - Checking in on this PR. Is the correct status still hold-off? Thanks.

**Comment by armanio123** [comment]
I think we should still hold off but I'm not entirely sure of the process. This feature just got released to Visual Studio preview channel.

**Comment by Mikejo5000** [comment]
If it's not a public preview, we should hold off. It it goes out in a public preview, we can doc it, but it needs to be stated that the feature is only available starting in Visual Studio 2019 Preview X.

**Comment by jborsecnik** [comment]
#hold-off

**Comment by armanio123** [comment]
hi everyone, the new version has release as of Jul/24 so this is good to be merged. Thanks for the patience.

**Comment by Mikejo5000** [comment]
#sign-off Thanks!

**Pull request merged**